src/components/pages/Home.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please do not disable linters this way. Linters help you keep your code clean, tidy, uniform and of high quality 👍